server/middleware/fetchFrontendComponentMiddleware.ts

```diff
@@ -0,0 +1,28 @@
+import { Router } from 'express'
+import { DataAccess } from '../data'
+import logger from '../../logger'
+import config from '../config'
+
+export default function setUpFrontendComponents({ frontendComponentApiClient }: DataAccess): Router {
```

Comment:
I'm not sure about the structure of this function 🤔 
We dont have a unit test for it, and I'm not sure how easy it would be to write one because it news up the `Router` internally
I wonder if there's a better/different way of structuring this .... 🤔

Reply:
Currently rewriting this logic with a service